doc/modules/ROOT/pages/configuration.adoc

```diff
@@ -39,6 +39,7 @@ SmallRye JWT supports many properties which can be used to customize the token p
 [cols="<m,<m,<2",options="header"]
 |===
 |Property Name|Default|Description
+|smallrye.jwt.verify.secretkey|none|Secret key supplied as a string.
```

Comment:
Will any other properties be ignored if this is set?

Reply:
@MikeEdgar Hi Mike, right now the order is as follows, if the MP JWT public key is set, it is picked up, otherwise this one is checked and finally, the keys from locations are loaded. At the moment there is no code there, for example, failing the build if someone sets both public and secret keys, in this case the secret key will be ignored.
Let me add some warnings if several properties pointing to the keys are set

Reply:
Thanks, it just occurred to me because `smallrye.jwt.verify.key.location` below mentions that `mp.jwt.verify.publickey.location` is ignored if both are set.

Reply:
Hi @MikeEdgar, I've added some logging, keeping the ordering in mind. The secret key will be checked only if the public key is not set,  and in both cases, if other key properties are set, there will be a warning.

That code requires some good cleanup though in general :-).

If you'd like something else to be updated right now, then just let me know, thanks

Reply:
Latest commit looks good to me, thanks @sberyozkin . Nothing more from me :smile: